Appcues Events updates [DOC-389] (#2243)
segmentio/segment-docs: src/connections/destinations/catalog/appcues/index.md

title: Appcues Destination
hide-cmodes: true
---
[Appcues](https://www.appcues.com/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners) adds an experience layer to your product so you can build user onboarding, NPS surveys, or feature announcements in minutes instead of weeks. The Appcues JavaScript Destination is open-source. You can browse the code [on GitHub](https://github.com/appcues/analytics.js-integration-appcues).

> To contribute to this page, visit [Segment’s documentation repository](https://github.com/segmentio/segment-docs/blob/develop/CONTRIBUTING.md).

Reviewer: Let's remove this as a one-off and explore how we can better add this CTA to all docs.

3. In the destination settings, enter your `Appcues Id` (for client-side integration functionality) and/or your `API Key` (for server-side integration functionality) from the [Appcues account page](https://my.appcues.com/account).

### Server

As an alternative to a traditional JavaScript implementation, Appcues offers a server-side destination with Segment.

You may find the server-side destination useful if you'd like to send user profile or event data to Appcues from another Segment partner service. You can use the server-side destination alongside the JavaScript destination, which you may find preferable to routing all data through the JavaScript destination.

As with the JavaScript destination, you can segment and target user profile and event data received through the Appcues server-side Segment destination.

For example, using the server-side destination, you can direct customer profile and event data from a CRM tool into Appcues. You can then use the directed data for content targeting and user segmentation in the Appcues content editor, alongside data from Segment's `analytics.js` destination.

## Page

> Refer to the Segment Spec for information about the [Page method](/docs/connections/spec/page/). The following represents an example `page` call:

Reviewer: For a long time, the template that our partners used to draft their docs used this "If you're not familiar with..." verbiage. I have a strong dislike for it, and your rewrite is a big improvement.

I'm tempted to do a site wide search / replace, but that's probably risky :)

Author: I didn't even know there was a template!  But that explains why that phrase kept reappearing.

```javascript
analytics.page();
```

> Each time the page changes, Appcues checks to see if a user qualifies for an experience. When you first call `page` using `analytics.js`, `Appcues.start` checks if there are any current flows for the user and loads them, if necessary.

Reviewer: I don't think we need `analytics.js` in code format here.

Author: I've reverted it to Analytics.js.

});
```

> When you use `identify` on `analytics.js`, Segment call `Appcues.identify`. This is the preferred method of using and targeting on user properties.

Reviewer: Same as above wrt `analytics.js`.

```suggestion
When you call `identify` with Analytics.js, Segment calls `Appcues.identify`. This is the preferred method of using and targeting on user properties.
```

> To get the most out of Appcues, you should send as much user data as possible in the `identify` call. Properties personalize content and target experiences to specific users. Most Appcues customers send properties that fall into a few groups:

Reviewer: ```suggestion
To get the most out of Appcues, you should send as much user data as possible in the `identify` call. Properties personalize content and target experiences to specific users. Most Appcues customers send properties that fall into one of the following groups:
```

To enable this feature, go to the Integrations Settings in Appcues and click "Activate" under the Segment integration.

> After you activate the Segment integration, Appcues automatically forwards these events to your Segment workspace. The events will appear in the same [Source Debugger](/docs/connections/sources/debugger/).

Reviewer: ```suggestion
After you activate the Segment integration, Appcues forwards these events to your Segment workspace. The events will appear in the same [Source Debugger](/docs/connections/sources/debugger/).
```

### Whitelisted Domains

By default, Appcues targets based on the URL's path.

For example, if you create an Appcues experience and target it to `/integrations`, it appears wherever the embed script is installed with that URL path, like `appcues.com/integrations` and `my.appcues.com/integrations`.

> If you've installed the `analytics.js` script on multiple domains (like a marketing site and your web app), you should use Appcues-whitelisted domains when targeting your experience.

Reviewer: ```suggestion
If you've installed the Analytics.js script on multiple domains (like a marketing site and your web app), you should use Appcues-whitelisted domains when targeting your experience.
```

## Delete

> When you trigger a user deletion using [Segment's Privacy features](/docs/privacy/user-deletion-and-suppression/), Segment forwards an actionable delete notification to Appcues. Refer to the Appcues documentation for information on [how Appcues handles deletion requests](https://docs.appcues.com/article/443-gdpr-deletion-api).

Reviewer: ```suggestion
When you trigger a user deletion using [Segment's Privacy features](/docs/privacy/user-deletion-and-suppression/), Segment forwards an actionable delete notification to Appcues. Refer to the Appcues documentation for information on [how Appcues handles deletion requests](https://docs.appcues.com/article/443-gdpr-deletion-api){:target="_blank"}.
```
